lib/rbUtil.js

```diff
@@ -173,6 +173,23 @@ rbUtil.parseETag = function(etag) {
     }
 };
 
+/**
+ * Copies forwarded headers from restbase to request.
+ * If the same header was already set it takes precedence over
+ * the forwarded header.
+ */
```

Comment:
@gwicke b was my main reason for making manual copies. External requests might be added in the future, and if we forget about this rule - we expose a disastrous hole in security, bc we would basically send complete user login information to a third party service. I can change this, but I thought it would be better to be more secure with a cost of a bit uglier code. Anyway this whole thing is quite ugly and it would be changed when we have a generic auth service.